packages/canary-react/src/component-tests/IcTreeView/IcTreeView.cy.tsx

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React, { ReactElement } from "react";
+import { mount } from "cypress/react";
+import { IcTreeView, IcTreeItem } from "../../components";
```

Comment:
Could expand Cypress tests to cover more of the acceptance criteria (navigating and selecting with mouse and with keyboard, hover state, onClick event etc.). Not sure if this would be better raised separate ticket raised to add them at a later date though.

Reply:
I'll add them in this PR in case anything happens to break them then it'd be captured straight away